File: src/main/java/com/dh/ondot/member/domain/event/UserRegistrationEvent.java
package com.dh.ondot.member.domain.event;

import com.dh.ondot.member.domain.enums.OauthProvider;

public record UserRegistrationEvent(
        Long memberId,
        String email,
        OauthProvider oauthProvider,
        Long totalMemberCount,
        String mobileType
) {
}

File: src/main/kotlin/com/dh/ondot/member/domain/event/UserRegistrationEvent.kt
package com.dh.ondot.member.domain.event

import com.dh.ondot.member.domain.enums.OauthProvider

data class UserRegistrationEvent(
    val memberId: Long,
    val email: String?,
    val oauthProvider: OauthProvider,
    val totalMemberCount: Long,
    val mobileType: String?,
)
